tests/test-sessions/test-sessions-common/src/test/java/org/eclipse/jetty/server/session/SessionListenerTest.java

```diff
@@ -115,12 +128,12 @@ public void testListenerWithInvalidation() throws Exception
             }
             finally
             {
-                client.stop();
+                LifeCycle.stop(client);
```

Comment:
What is the benefit of doing this, rather than simply client.stop()? Same comment for LifeCycle.stop(server) vs just server.stop()?

Reply:
Interesting. I wonder if tests _should_ fail if the server/client can't be stopped? Is it diagnostic of some greater problem?